siliconcompiler/toolscripts/ubuntu26/install-gtkwave.sh

```
@@ -35,6 +35,9 @@ fi
 cd gtkwave3-gtk3
 
 ./autogen.sh
-./configure --enable-gtk3 $args
+# GCC 14+ (Ubuntu 26.04) promotes -Wincompatible-pointer-types to a hard error,
+# which breaks GTKWave's GTK3 callback casts. Downgrade it back to a warning.
+CFLAGS="-Wno-error=incompatible-pointer-types -Wno-error=int-conversion" \
+    ./configure --enable-gtk3 $args
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)